reports/src/main/java/io/jmix/reports/app/StrategyGsonBuilder.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.reports.app;
+
+import com.google.gson.ExclusionStrategy;
+import com.google.gson.FieldAttributes;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Arrays;
+
+public class StrategyGsonBuilder {
```

Comment:
A very common class name that doesn't explain what the class does. What strategy does it build? As I see, you use only ignoreMetadata serialization policy. Maybe it's worth thinking about removing the SerializationPolicy enum at all and rename the class into something like `MetadataIgnoringGsonBuilder`?